packages/types/config/index.d.ts

```diff
@@ -32,6 +34,7 @@ export interface NuxtOptions extends Configuration {
   buildDir: string
   buildModules: NuxtOptionsModule[]
   cli: NuxtOptionsCli
+  components: boolean | Array<string | ComponentsDir>
```

Comment:
There is also object format support (which may be used later on to provide more options

Reply:
Ideally the options would be completely defined in `@nuxt/components` and just imported in `@nuxt/types`. I will propose a PR to do that if that sounds good to you.

Reply:
Indeed. Just published types for both :see_no_evil: Updating deps in a sec.

Reply:
#7621